libraries/AP_HAL_ChibiOS/hwdef/BlitzF745/README.md

```diff
@@ -0,0 +1,90 @@
+# iFlight BLITZ Whoop F7 AIO Flight Controller
+
+https://shop.iflight.com/electronics-cat27/BLITZ-F745-Flight-Controller-Pro2141
+
+The Blitz F7 is a flight controller produced by [iFlight](https://shop.iflight-rc.com/).
+
+## Features
+
+ - MCU: BGA-STM32F745, 216MHz
+ - Gyro: ICM42688
+ - 32Mb Onboard Flash
+ - BEC output: 5V 2.5A
+ - Barometer: DPS310
+ - OSD: AT7456E
+ - 6 UARTS: (UART1, UART2, UART3, UART4, UART5, UART6)
+ - I2C for external compass.
+ - 9 PWM outputs (8 motor outputs and 1 LED output)
+
+## Pinout
+
+![BLITZ F745 Board](blitz_f7_pinout.jpg "BLITZ F745")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+|Name|Pin|Function|
+|:-|:-|:-|
+|SERIAL0|COMPUTER|USB|
+|SERIAL1|RX1/TX1|UART1 (DJI connector)|
+|SERIAL2|TX2/RX2|UART2 (RX, DMA-enabled)|
+|SERIAL3|TX3/RX3|UART3 (GPS, DMA-enabled)|
+|SERIAL4|TX4/RX4|UART4|
+|SERIAL5|TX5/RX5|UART5|
+|SERIAL6|RX6|UART6 (ESC telemetry)|
```

Comment:
```suggestion
|SERIAL6|RX6|UART6 (R6 only pin,ESC telemetry)|
```

Reply:
Already shows RX6 only

Reply:
okay, did not see it in that table format